.github/workflows/build.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       matrix:
         os: [ubuntu-18.04]
         python-version: [3.7,3.8,3.9]
-        test-folder : ['tests.tests_iotools']
+        test-folder : ['tests.tests_iotools', 'tests']
```

Comment:
Would just leaving "tests" here work? since tests_iotools is in tests?

Reply:
I'll check

Reply:
Yes it works

```
Run $CONDA/bin/pytest
============================= test session starts ==============================
platform linux -- Python 3.9.5, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
rootdir: /home/runner/work/ioSPI/ioSPI
plugins: cov-2.12.1
collected 18 items

tests/test_fourier.py .......                                            [ 38%]
tests/test_transfer.py ...                                               [ 55%]
tests/test_iotools/test_atomic_models.py ...                             [ 72%]
tests/test_iotools/test_cryo_dataset.py .....                            [100%]

============================== 18 passed in 5.53s ==============================
```